### PR TITLE
chore(dev-deps): minor updates

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[][demo] |
:---:|

## 🧰 Changes

Updates a bunch of dev deps.

    "hast-util-to-string": "^1.0.4",
    "@babel/core": "^7.14.8",
    "@babel/plugin-proposal-class-properties": "^7.14.5",
    "@babel/plugin-proposal-object-rest-spread": "^7.14.7",
    "@babel/plugin-proposal-optional-chaining": "^7.14.5",
    "@babel/preset-env": "^7.14.8",
    "@babel/preset-react": "^7.14.5",
    "@commitlint/cli": "^13.1.0",
    "@commitlint/config-angular": "^13.1.0",
    "@commitlint/config-conventional": "^13.1.0",
    "@readme/eslint-config": "^6.0.0",
    "babel-jest": "^27.0.6",
    "codemirror": "^5.62.2",
    "core-js": "^3.15.2",
    "eslint": "^7.31.0",
    "jest": "^27.0.6",
    "jest-image-snapshot": "^4.5.1",
    "prettier": "^2.3.2",
    "puppeteer": "^10.1.0",
    "rdme": "^4.0.0",
    "regenerator-runtime": "^0.13.9",
    "semantic-release": "^17.4.4",
    "string.prototype.trimend": "^1.0.4",
    "terser-webpack-plugin": "^5.1.4",
    "webpack": "^5.47.0",
    "webpack-merge": "^5.8.0"

### Hacks

There's some issue on Github Actions:

```
  Run npm ci
  npm ci
  sudo apt install fonts-noto-color-emoji
  make emojis
  shell: /usr/bin/bash -e {0}
npm ERR! string.prototype.trimend not accessible from es-abstract
```

Adding `strting.prototype.trimend` seems to workaround it?!

### CODE REVIEW

File: __tests__/components/test.js [hunk 1]
```diff
@@ -148,21 +148,27 @@ describe('Components', () => {
     const box = wrap.find('.lightbox').at(0);
 
     img.simulate('click');
+    // eslint-disable-next-line jest-dom/prefer-to-have-attribute
     expect(box.getDOMNode(0).hasAttribute('open')).toBe(true);
```
Comment (erunion) [inline __tests__/components/test.js:152]: Does `.toHaveAttribute('open')` not work with `.getDOMNode()`?

Comment (kellyjosephprice) [reply __tests__/components/test.js:152]: It did not. :shrug:

**Review by erunion — APPROVED** [review]

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.30.3-next.2</b>](/readmeio/markdown/releases/tag/v6.30.3-next.2)
<br>

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.30.3</b>](/readmeio/markdown/releases/tag/v6.30.3)
<br>

**Pull request merged**